jobs-tests/src/test/java/org/zowe/jobs/tests/AbstractJobsIntegrationTest.java

```diff
@@ -78,6 +79,13 @@ static Response submitJobJclString(String jclString) throws Exception {
     public static Response deleteJob(Job job) throws Exception {
         return RestAssured.given().when().delete(getJobPath(job));
     }
+    
+    public static Response modifyJob(Job job, String command) throws Exception {
+        JsonObject body = new JsonObject();
+        body.addProperty("command", command);
+        Response response = RestAssured.given().contentType("application/json").body(body.toString()).when().put(getJobPath(job));
+        return response;
+    }
```

Comment:
It doesn't look like this is used in any other test, so should be pushed down into JobModifyIntegrationTest